dataInterface/items.js

```diff
@@ -8,16 +8,28 @@ module.exports.getAll = () => {
   return module.exports.items;
 }
 
-module.exports.getById = (itemId) => {
-  // TODO: complete writing this function
+module.exports.getById = (id) => {
+  let itemArr = module.exports.items;
+  let item = itemArr.find( element => element.id === id);
+  return item;
 }
 
 module.exports.deleteById = async (itemId) => {
     // TODO: complete writing this function
```

Comment:
We always want to keep comments up to date in our code, so these TODOs should be deleted once the specified work is done.